src/FormBuilder/resources/js/form-builder/src/components/sidebar/Primary/index.tsx

```diff
@@ -44,6 +48,7 @@ const tabs = [
                 <DonationConfirmation />
                 <FormGridSettings />
                 <EmailSettings />
+                {wp.hooks.applyFilters('givewp_form_builder_pdf_settings', '')}
```

Comment:
@glaubersilva ideally we would have a slot/fill here for add-ons to add their own settings - similar to our field settings slot/fill.  This is probably fine for now but would definately like to revisit this extension for 3.1

Reply:
Yep, due to the time I think is a good idea to do it later, would be nice to better understand how this slot/fill works.

Reply:
i'm back to thinking we should just make this a slot.  i would even be fine doing a pdf receipts specific slot.  reason being a filter can be modified and I don't think we want these settings to be modified with how complex they are.  I can help you with it

Reply:
Disregard this for now, i just realized we would still need a filter of some sort in order to render the Fill inside an add-on.  So, this is fine for now - i'll work on an api for this.